scripts/surgical_robotics_challenge/psm_arm.py

```diff
@@ -43,10 +43,10 @@
 #     \version   1.0
 # */
 # //==============================================================================
-from surgical_robotics_challenge.kinematics.psmIK import *
+from surgical_robotics_challenge.kinematics.psmKinematics import *
 from surgical_robotics_challenge.utils.joint_errors_model import JointErrorsModel
 from surgical_robotics_challenge.utils import coordinate_frames
-
+import rospy
```

Comment:
Do we need rospy? If not, let's remove this import.

Reply:
In line #216-219, rospy was used for the function "_execute_trajectory(...)". Therefore, I believe the rospy import is required.